include/pdal/Utils.hpp

```diff
@@ -149,17 +149,17 @@ namespace Utils
 
     inline std::string tolower(const std::string& s)
```

Comment:
I'll have to pull it back out to reproduce the actual warning/error that led me to look at this. The stackoverflow link in the commit message has some commentary on this approach, having to do in part with signed vs. unsigned char's. The 'X' bit is just me initializing the output string to be the proper length. Maybe I could have resized instead. It must be the right size before using transform though, or bad things happen.